layout.tt

```diff
@@ -24,10 +24,23 @@ BLOCK navigationLink %]
 
 -%]
 
+[% DEFAULT
+    title = ''
+    description = 'Nix is a tool that takes a unique approach to package management and system configuration. Learn how to make reproducible, declarative and reliable systems. '
+%]
+
 <html lang="en" class="without-js">
 
 <head>
-  <title>[% title %]</title>
+
+  [% IF title != "" %]
+  <title>[% title %] | Nix &amp; NixOS </title>
+  [% ELSE %]
+  <title>Nix &amp; NixOS | Reproducible builds and deployments</title>
```

Comment:
```suggestion
  <title>[% title %] | Nix </title>
  [% ELSE %]
  <title>Nix | Reproducible builds and deployments</title>
```

Except for the odd announcement, the current web site is exclusively about Nix and clearly not about NixOS. We have unambiguous evidence that beginners have a hard time distinguishing the different software components and their purpose and role. It of course needs more work to introduce the other projects and systems, but let's keep it focused on Nix for now.

Reply:
I'm leaving both in the title as we are also offering the download of NixOS.

And meanwhile we'll figure out the messaging on the website to introduce both Nix and NixOS ideas.